composer.json

```diff
@@ -5,14 +5,15 @@
     "homepage": "https://b13.com",
     "license": ["GPL-2.0-or-later"],
     "require": {
-        "typo3/cms-backend": "^10.4.18 || ^11.5",
-        "doctrine/dbal": "~2.13.1"
```

Comment:
I think we should keep the explicit dependency to doctrine/dbal (with ~2.13.1 || ~3.4)